### ISSUE TITLE
[ValueError@binaryview.py:9122] ValueError: Couldn't find target for type


### ISSUE THREAD

**Issue by mm4rks** [fkie-cad/dewolf#251]
# What happened?

```python
  File &quot;/opt/dewolf/decompiler/util/bugfinder/bugfinder.py&quot;, line 164, in iter_function_reports
    task_result = self.decompile(function, options)
  File &quot;/opt/dewolf/decompile.py&quot;, line 50, in decompile
    task = self._frontend.create_task(function, task_options)
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/frontend.py&quot;, line 143, in create_task
    raise e
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/frontend.py&quot;, line 130, in create_task
    cfg = self._extract_cfg(function.function, options)
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/frontend.py&quot;, line 162, in _extract_cfg
    return parser.parse(function)
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/parser.py&quot;, line 34, in parse
    index_to_BasicBlock[basic_block.index] = BasicBlock(basic_block.index, instructions=list(self._lift_instructions(basic_block)))
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/parser.py&quot;, line 77, in _lift_instructions
    if lifted_instruction := self._lifter.lift(instruction):
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/lifter.py&quot;, line 28, in lift
    if pseudo_expression := handler(expression, **kwargs):
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/handlers/assignments.py&quot;, line 51, in lift_assignment
    self._lifter.lift(assignment.src, parent=assignment),
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/lifter.py&quot;, line 28, in lift
    if pseudo_expression := handler(expression, **kwargs):
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/handlers/constants.py&quot;, line 76, in lift_constant_pointer
    return self._lifter.lift(variable, view=view, parent=pointer)
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/lifter.py&quot;, line 28, in lift
    if pseudo_expression := handler(expression, **kwargs):
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/handlers/globals.py&quot;, line 30, in lift_global_variable
    initial_value=self._get_initial_value(variable, view)
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/handlers/globals.py&quot;, line 39, in _get_initial_value
    return self._lifter.lift(view.get_data_var_at(variable.value), view=view)
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/lifter.py&quot;, line 28, in lift
    if pseudo_expression := handler(expression, **kwargs):
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/handlers/globals.py&quot;, line 30, in lift_global_variable
    initial_value=self._get_initial_value(variable, view)
  File &quot;/opt/dewolf/decompiler/frontend/binaryninja/handlers/globals.py&quot;, line 41, in _get_initial_value
    return Constant(variable.value)
  File &quot;/opt/binaryninja/python/binaryninja/binaryview.py&quot;, line 9229, in value
    return self._accessor.value
  File &quot;/opt/binaryninja/python/binaryninja/binaryview.py&quot;, line 9082, in value
    return self._value_helper(self.type, self.view.read(self.address, len(self.type)))
  File &quot;/opt/binaryninja/python/binaryninja/binaryview.py&quot;, line 9122, in _value_helper
    raise ValueError(&quot;Couldn&#x27;t find target for type&quot;)

```

Error class `ValueError@binaryview.py:9122` contains **1251** cases.

# How to reproduce?

```bash
python decompile.py a776c4c370c289d13498c053dd7b1bfcbb342560d3fa60fff4b7d75feca45746 sub_3610 --debug
```

sample: [a776c4c370c289d13498c053dd7b1bfcbb342560d3fa60fff4b7d75feca45746](https://bugfinder.seclab-bonn.de/download/a776c4c370c289d13498c053dd7b1bfcbb342560d3fa60fff4b7d75feca45746)
dewolf commit: 5566666af0d3271618e758808fbc7be92be02e1d
Binaryninja version: `3.4.4271`

**Comment by NeoQuix** [comment]
Fixed with #233 
Need to talk about `NamedTypeReferences` because I'm a bit confused

**Comment by NeoQuix** [comment]
Fixed temporary with `unknown value` + bninja upstream for 3.5
